js/modules/loki_app_dot_net_api.js

```diff
@@ -89,6 +93,13 @@ class LokiAppDotNetServerAPI {
     })();
   }
 
+  async close() {
```

Comment:
unregisterChannel() is already existing

Reply:
Good point. However `unregisterChannel` takes a channel id then find the corresponding channel, then closes it and removes it from the list. Feels like in my case, since I want to close all channels, iterating over the channels is what I want instead of looking up every id separately. Does that make sense?
I'm happy to rename `close()` to `unregisterAllChannels` to be more consistent.